.github/workflows/stale.yaml

```diff
@@ -0,0 +1,28 @@
+name: Mark stale PRs and close rotten PRs
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+permissions:
+  contents: read
+
+jobs:
+  stale:
+
+    permissions:
+      pull-requests: write  # for actions/stale to close stale PRs
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v8
```

Comment:
can we pin this to the commit hash instead per github security guidelines?  https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions